modules/integration/tests-common/test-utils/src/main/java/org/wso2/carbon/ml/integration/common/utils/MLHttpClient.java

```diff
@@ -461,6 +467,8 @@ public String getModelName(CloseableHttpResponse response) throws MLHttpClientEx
             return responseJson.getString("name");
         } catch (IOException e) {
             throw new MLHttpClientException("Failed to get the name of model" , e);
+        } catch (JSONException e) {
+            throw new MLHttpClientException("Failed to get the name of model" , e);
         }
```

Comment:
We're returning the same exception message here right? Hence, can we catch Exception only without catching all different exceptions? There's a typo in the message. /s/Faile/Failed